crates/but-cli/src/command/mod.rs

```diff
@@ -117,4 +119,15 @@ pub mod stacks {
         let ctx = CommandContext::open(&project, AppSettings::default())?;
         debug_print(stack_branches(id.to_string(), &ctx))
     }
+
+    pub fn branch_commits(id: &str, name: &str, current_dir: &Path) -> anyhow::Result<()> {
+        let project = project_from_path(current_dir)?;
+        let ctx = CommandContext::open(&project, AppSettings::default())?;
+        let local_and_remote =
+            stack_branch_local_and_remote_commits(id.to_string(), name.to_string(), &ctx);
+        debug_print(local_and_remote)?;
```

Comment:
👀

Reply:
And it's called `debug_*` because it invokes the `std::fmt::Debug` version of the Rust formatting system, basically the Rust object notation (RON), as the cheapest possible way to get an idea of what functions produce.

`but-cli` is used to bring new V3 implementations into the real world for the first time, and sometimes help with measuring performance in a non-interactive way.